### ISSUE TITLE
Weapon Ammo Refill Exploit

### ISSUE THREAD

**Issue by hashnimmo** [dreamstalker/rehlds#544]
So I have my own server which I removed some files in it that I thought that wasn't necessary. But the server works without any errors. I have installed both ReHLDS and ReGameDLL in my server and they both seems to be injected to the server properly.

One day a player that played in my server reported that he can empty any type of a gun and refill it's ammo for free and instantly by dropping and picking up that weapon quickly while standing near a wall, this has to be done standing close to the wall and facing the direction where the wall stands so as soon as he drops the weapon he can automatically pick it up because the weapon is not getting dropped far away from him. By doing this quickly about 3 times the weapon's ammo get refilled. I have tried this myself and it worked.

Maybe this has something to do with me removing some of my essential server files. I'm sure that none of my plugins doing this. So I thought I should report this here if someone has missed it.

**Comment by In-line** [comment]
I wasn't able to reproduce this. Can you confirm that bug still exists?

**Comment by hashnimmo** [comment]
Nope. I think it had something to do with those missing server files that I deleted recently, but with the full server package, this issue is gone now.
